estela_cli/context.py

```diff
@@ -1,15 +1,21 @@
+import yaml
 import click
 import requests
 
+from getpass import getpass
 from estela_cli.login import env_login, yaml_login
+from estela_cli.estela_client import EstelaClient
 from estela_cli.utils import (
     get_estela_auth,
+    update_estela_auth,
     get_estela_settings,
+    get_estela_config,
     get_host_from_env,
     get_username_from_env,
     get_password_from_env,
+    get_estela_config_path,
 )
-from estela_cli.templates import ESTELA_AUTH_NAME, OK_EMOJI, BAD_EMOJI
+from estela_cli.templates import ESTELA_AUTH_NAME, ESTELA_CONFIG_NAME, OK_EMOJI, BAD_EMOJI
 
 
 SHORT_HELP = "Show your current context"
```

Comment:
We should update the description of the command here to also indicate that it serves to update/change the context